spec/services/globus_digest_generator_spec.rb

```diff
@@ -68,7 +68,8 @@
 
     before do
       FileUtils.rm_rf('tmp/globus/test')
-      FileUtils.cp_r('spec/fixtures', 'tmp/globus/test')
+      FileUtils.mkdir_p('tmp/globus/test')
+      FileUtils.cp_r('spec/fixtures/.', 'tmp/globus/test')
```

Comment:
Per `FileUtils` documentation:

> `# If you want to copy all contents of a directory instead of the`
> `# directory itself, c.f. src/x -> dest/x, src/y -> dest/y,`
> `# use following code.`
> `FileUtils.cp_r 'src/.', 'dest'     # cp_r('src', 'dest') makes dest/src,`
> `                                   # but this doesn't.`